frontend/src/pages/SearchUILandingPage.js

```diff
@@ -109,7 +109,7 @@ class SearchUILandingPage extends React.Component<SearchUILandingPageDefaultProp
         docs = 'one document';
         break;
       default:
-        docs = `${this.state.numDocuments.toLocaleString()} documents`;
+        docs = `${this.state.numDocuments.toLocaleString()} documents - HI!`;
```

Comment:
Um...?

Reply:
Removed - this was left over from something unrelated and slipped through my cleanup.

Reply:
I assumed it was a sort of "Are my changes showing up here?" sort of thing...  😈